services/movie.py

```diff
@@ -0,0 +1,39 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None, actors_ids: list[int] = None
+) -> QuerySet[Movie]:
+    queryset = Movie.objects.all()
+    if genres_ids and actors_ids:

```

Comment:
Please simplify it for better readability to keep only the code starting from line 14.